Lib/test/test_future.py

```diff
@@ -294,6 +295,15 @@ def test_fstring_debug_annotations(self):
         self.assertAnnotationEqual("f'{x=!a}'", expected="f'x={x!a}'")
         self.assertAnnotationEqual("f'{x=!s:*^20}'", expected="f'x={x!s:*^20}'")
 
+    def test_infinity_numbers(self):
+        inf = "1e" + repr(sys.float_info.max_10_exp + 1)
+        infj = f"{inf}j"
+        self.assertAnnotationEqual("1e1000", expected=inf)
+        self.assertAnnotationEqual("1e1000j", expected=infj)
+        self.assertAnnotationEqual("3+1e1000j", expected=f"(3+{infj})")
+        self.assertAnnotationEqual("(1e1000, 1e1000j)", expected=f"({inf}, {infj})")
+        self.assertAnnotationEqual("'inf'")
+        self.assertAnnotationEqual("('inf', 1e1000, 'infxxx', 1e1000j)", expected=f"('inf', {inf}, 'infxxx', {infj})")
```

Comment:
Please test minus infinity, like -1e1000. I expect to get something like -(inf). rather than (-inf).

Reply:
I'll add the test but current version gives this which I believe correct;
```py
>>> a: -1e1000
>>> __annotations__.get("a")
'-1e309'
```

Reply:
So add a test for it, to ensure that we don't regress :-)

Reply:
added